src/api/apiClient.ts

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable no-console */
+import axios, { AxiosError } from 'axios'
+
+export const BASE_URL = 'https://api.devcycle.com/'
+// export const BASE_URL = 'http://localhost:4001/'
+export const AUTH_URL = 'https://auth.devcycle.com/'
+
+export const apiClient = axios.create({
+    baseURL: `${BASE_URL}/v1/api`,
```

Comment:
is this correct? our api doesn't have `/api` in the path

Reply:
yeah this was not properly tested since we were passing BASE_URL and is fixed now